deployment/local-jvm/bash-scripts/attemptBulkTest.sh

```diff
@@ -1,10 +1,6 @@
 #!/usr/bin/env bash
 
-FILE=example-model/target/example-model-*-exec.jar
+FILE=example-model/target/example-model-*-exec.jar || echo "Cannot find example-model-<version>-exec.jar - have you run 'mvn install'?"
```

Comment:
This else never gets called when I run after a 'mvn clean'. instead I get the following error "Error: Unable to access jarfile example-model/target/example-model-*-exec.jar"